### ISSUE TITLE
useRestyle's filterRestyleProps function throwing error

### ISSUE THREAD

**Issue by niranjannitesh** [Shopify/restyle#160]
```ts
TypeError: undefined is not an object (evaluating 'omitPropertiesMap.variant')
```

I believe this line (https://github.com/Shopify/restyle/blob/master/src/hooks/useRestyle.ts#L17) should be
```diff
-  const props = omitPropertiesMap.variant
+  const props = omitPropertiesMap?.variant
```

**Comment by sampsonjoliver** [comment]
Getting this as well for calling `useRestyle` with any style function created through `createRestyleFunction`.

**Comment by fortmarek** [comment]
Could you provide reproduction steps? I'm surprised this happens since Typescript doesn't complain.

**Comment by niranjannitesh** [comment]
closed this issue as I am not facing this issue anymore, seems like it was a migration issue from my side.
